common/protocol/src/main/java/org/sdo/iotplatformsdk/common/protocol/security/keyexchange/AsymmetricKeyExchange.java

```diff
@@ -93,7 +93,7 @@ private UUID getUuid() {
 
   // xB is B, enciphered with the owner's key.
   public void setXb(ByteBuffer xb) {
-    final ByteBuffer buf = cipherFactory.buildDecipher(CIPHER, xb, getUuid());
+    final ByteBuffer buf = cipherFactory.buildDecipher(xb, getUuid());
```

Comment:
This was legacy, and was only used by an implementation of AsymKexCodec where the asymmetric keys were present with the owner, and constant CIPHER used to denote the algorithm to be applied by the owner.
Since in iot-platform-sdk, the keys are present at the OCS, and not with the owner (OPS), there's no need for the CIPHER algorithm parameter in OPS. The value of the CIPHER is a constant at OCS, since the operation is fixed and is performed there.